plugins/inputs/win_services/README.md

```diff
@@ -0,0 +1,76 @@
+# Telegraf Plugin: win_services
+Input plugin to report Windows services info: service name, display name, state, startup mode
```

Comment:
Don't list the tags/fields here, so we won't need to keep this updated if we add anything in the future.